README.md

```diff
@@ -57,15 +57,24 @@ $ python tools/symlink_common.py .
 
 Adding contributors
 -------------------
-
-``contributors.json`` contains a JSON dictionary of all contributors who have
-signed the scitools CLA. It is the definitive source of such contributors.
-It is where the [CLA bot](https://github.com/SciTools-incubator/scitools-cla-checker)
-looks to determine if a CLA is required.
-If you've signed the CLA and aren't yet on this list, please consider submitting a PR
-to speed the process up. 
-A [rendered view](https://scitools.org.uk/signed_cla.html) of the list is also
-available.
+Our contributors list, and Contributor License Agreement ([the "CLA"](https://scitools.org.uk/cla/v4)), are currently in the process of change.\
+There is a long list of earlier SciTools contributors who signed our original CLA based explictly on [the LGPL3 License](<https://www.gnu.org/licenses/lgpl-3.0.html>).\
+However, we are now attempting to re-license all the SciTools projects under a new CLA that will allow us in future to adopt _any_ OSI license,
+as explained [here](https://github.com/SciTools/scitools.org.uk/wiki/Contributors-License-Agreement-processing-details).
+
+This process will only be complete when all previous contributors have signed the new terms.
+Until then, we respect the rights of all the original contributors, in addition to those on the new list.
+
+Details of the former, original Iris CLA and contributors list were as follows :
+
+    ``contributors.json``, in this repo, contains a JSON dictionary of all contributors who have
+    signed the scitools CLA. It is the definitive source of such contributors.
+    It is where the [CLA bot](https://github.com/SciTools-incubator/scitools-cla-checker)
+    looks to determine if a CLA is required.
```

Comment:
The CLA bot must also be looking at the new list:
https://script.google.com/macros/s/AKfycbwqZ0JLlOKJEcvYUYFXo77o6nux9tSHvOJISUnk3yHi-Bk0Qanz/exec.

Reply:
Yes, I see it also [reads the old list](https://github.com/SciTools-incubator/scitools-cla-checker/blob/master/scitools_cla_checker/update_pr.py#L202), so for now it combines both.

I will add something to make this clearer.